doc/source/mast_data_products/pipeline-astrometric-calibration-description.rst

```diff
@@ -656,6 +656,67 @@ these computations were being performed in based up to the parent directory to
 replace the previously updated versions of the input files.  This entire sub-directory
 then gets deleted, unless the processing was being run in debug mode.
 
+Fit quality and selection of WCS solution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The primary WCS selected will depends on several factors. These include the ``fit_rms``, 
+and whether a solution is deemed *compromised* (discussed below). The ``fit_quality`` is an integer value 
+between 1 and 5, where 1 is the best fit and 5 is the worst. As *a posteriori* WCS solutions 
+are fit with different catalogs (e.g. GAIADR3, GAIADR2, etc.), and different fit geometries 
+(e.g. rscale, shift, etc.), the code will only adopt the new solution if it has a 
+fit_quality less than or equal to the previous value AND a lower fit_rms. The fit_quality 
+value is determined as follows:
+
+  * 1 = Valid solution with fit_rms < 10 mas
+
+  * 2 = Valid but compromised solution with fit_rms < 10 mas
+
+  * 3 = Valid solution with fit_rms >= 10 mas
+
+  * 4 = Valid but compromised solution with fit_rms >= 10 mas
+
+  * 5 = No valid solution
+
+  * -1 = Alignment code failed altogether. 
+
+
+There are several ways in which a WCS solution can be considered compromised. 
+These manifest as a series of checks done in the align.py function 
+*determine_fit_quality()*. Checks must return True, otherwise the solution is 
+considered compromised. These checks include:
+  
+  * **consistency_check**: Ensures that the 
+    combination of the standard deviation of the X and Y shifts is less than one 
+    of two values, whichever is larger. The limit is either the value defined 
+    in align_pars["determine_fit_quality"]["MAX_FIT_RMS"] (default is 20 pixels) 
+    or the mean of the *fit_rms* values for all observations, whichever is larger. 
+
+
+  * **large_rms_check**: Ensures that both the *fit_rms* and the *max_rms* are less 
+    than the *max_fit_limit* (default is 150). The *fit_rms* is the RMS between the 
+    astrometric catalog and the image alignment sources for an individual 
+    exposure. The *max_rms* is the maximum of the *fit_rms* values for all of the 
+    exposures in the association.
+
+
+  * **radial_offset_check**: Ensures that the image alignment source magnitudes 
+    and astrometric catalog magnitudes have a *fit_rms* greater than 0.5.
```

Comment:
This description is not correct.  This check is not based on magnitudes.  After the actual radial offset is computed based upon, presumably, the shift in x and y between the image data and the catalog.  However, the radial_offset_check is then computed on an equation that I have no idea where it comes from.  We can only document this now, but this equation is strange, particularly as it computes a value to the power of 8!